src/components/timeline/README.md

```diff
@@ -1,12 +1,115 @@
 # Timeline
-The timeline component can be used to visualize a series of events, deadlines, or other time-relevant material.
+
+Show users chronologically ordered information
 
 ## When to use
-- Your website or app needs to display a series of time-related events/activities that require further explanation
 
-## When to consider something else
-- When communicating where users are in a process, use the Step Indicator
-- For normal lists, use ordered lists or unordered lists
+The timeline component should be used to structure chronologically relevant information in a visually meaningful way. This can include:
+
+- Displaying a series of time-related events/activities that require further explanation
```

Comment:
Since this list only has one item, would it be better to combine it all into one or two paragraphs?